core/src/main/resources/db/changelog/changesets/proxy_data_model.yaml

```diff
@@ -0,0 +1,19 @@
+databaseChangeLog:
+  - changeSet:
+      id: "proxy_data_model"
+      author: pegah
+      changes:
+        - createTable:
+            tableName: enrollee_relation
+            columns:
+              - column: { name: id, type: uuid, defaultValueComputed: gen_random_uuid(), constraints: { nullable: false, primaryKey: true } }
+              - column: { name: created_at, type: datetime, constraints: { nullable: false } }
+              - column: { name: last_updated_at, type: datetime, constraints: { nullable: false } }
+              - column: { name: participant_user_id, type: uuid, constraints: {#participant user id is the user who is the proxy but not the enrollee
+                nullable: false, foreignKeyName: fk_enrollee_relation_participant_user, references: participant_user(id)} }
+              - column:  { name: enrollee_id, type: uuid, constraints: { #enrollee id is the id of the user who is actually enrolled in the study
+                nullable: false, foreignKeyName: fk_enrollee_relation_enrollee, references: enrollee(id)}  }
+              - column: { name: portal_id, type: uuid, constraints: { nullable: false, foreignKeyName: fk_enrollee_relation_portal, references: portal(id) } }
```

Comment:
Sorry, this column is on the ERD, but I just realized that we don't need it.  And I see that you're not using it anywhere.  So let's remove it